test/webview/View.test.ts

```diff
@@ -318,11 +318,26 @@ describe('View', () => {
                         file: 'developer-tutorials/add-environment.md'
                     },
                     {
-                        title: 'Debug a smart contract',
-                        length: '30-40 mins',
-                        difficulty: 'moderate',
-                        description: 'Learn how to make iterative changes to your smart contract using the VS Code debugger',
-                        file: 'developer-tutorials/debug.md'
+                      title: 'Creating a new identity',
+                      length: '2-3 mins',
```

Comment:
I don't think these times are long enough. You should probably get Paul and Rob to follow the tutorials and ask them how long they took

Reply:
Fixed